app/celery/scheduled_tasks.py

```diff
@@ -222,6 +222,24 @@ def check_templated_letter_state():
             )
 
 
+@notify_celery.task(name="in-flight-to-inbox-sms")
+@statsd(namespace="tasks")
+def in_flight_to_inbox_sms():
+    _, notifications = sms_queue.expire_inflights()
+
+    for notification in notifications:
+        sms_queue.publish(notification)
```

Comment:
...and hence making these lines unnecessary. The beat process will not move the notifications. This would not guarantee required atomicity/transactional bound anyway, so we let the queue implementation do it all in one sweep in a Redis script that will execute within a necessary transactional bound (as far as Redis is concerned).